lib/ret_web/controllers/page_controller.ex

```diff
@@ -2,30 +2,24 @@ defmodule RetWeb.PageController do
   use RetWeb, :controller
   alias Ret.{Repo, Hub}
 
-  # Split the HTML file into two parts, on the line that contains HUB_META_TAGS, so we can add meta tags
-  # TODO DRY
-  @hub_html_chunks "#{Application.app_dir(:ret)}/priv/static/hub.html"
-                   |> File.read!()
-                   |> String.split("\n")
-                   |> Enum.split_while(&(!Regex.match?(~r/HUB_META_TAGS/, &1)))
-                   |> Tuple.to_list()
-
-  @smoke_hub_html_chunks "#{Application.app_dir(:ret)}/priv/static/smoke-hub.html"
-                         |> File.read!()
-                         |> String.split("\n")
-                         |> Enum.split_while(&(!Regex.match?(~r/HUB_META_TAGS/, &1)))
-                         |> Tuple.to_list()
-
   def call(conn, _params) do
     render_for_path(conn.request_path, conn)
   end
 
   def render_for_path("/", conn) do
-    render_file(conn, "#{get_file_prefix(conn)}index.html")
+    conn |> render_page("index")
   end
 
   def render_for_path("/link", conn) do
-    render_file(conn, "#{get_file_prefix(conn)}link.html")
+    conn |> render_page("link")
+  end
+
+  def render_for_path("/avatar-selector.html", conn) do
+    conn |> render_page("avatar-selector")
+  end
+
+  def render_for_path("/smoke-avatar-selector.html", conn) do
+    conn |> render_page("avatar-selector")
```

Comment:
`"smoke-avatar-selector"`?

Reply:
confusing but this is actually right -- think of the function arg as matching the route, but the `render_page` function doesn't require you include `smoke-`, it adds the `smoke-` prefix to the file key based on the `conn`